Software/app/sensors_lorawan/.project

```diff
@@ -1442,6 +1442,31 @@
 			<type>1</type>
 			<locationURI>PARENT-2-PROJECT_LOC/target/stm32wlxx_hal_driver/Inc/Legacy/stm32_hal_legacy.h</locationURI>
 		</link>
+		<link>
+			<name>lib/STM32WLxx_LoRaWAN/LoRaWAN/LmHandler/packages/FragDecoder.c</name>
+			<type>1</type>
+			<locationURI>PARENT-2-PROJECT_LOC/lib/STM32WLxx_LoRaWAN/LoRaWAN/LmHandler/Packages/FragDecoder.c</locationURI>

```

Comment:
'Packages' in the locationURI section of each of these should be 'packages' as STM32CubeIDE will not build. This is also for all the instances below in this file